fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/LoanWithWaiveInterestAndWriteOffIntegrationTest.java

```diff
@@ -140,7 +140,7 @@ public void checkClientLoanCreateAndDisburseFlow() {
     }
 
     @Test
-    public void checkClientLoan_WRITTEN_OFF() {
+    public void checkClientLoanWrittenOFF() {
```

Comment:
I don't think OFF is an abbreviation - so if we use CamelCase, let's do it for the whole word (even if CheckStyle would allow us not to)

Reply:
fixed.